### DISCUSSION TITLE
Disable all addons without having to uninstall and reinstall the extension

### DISCUSSION THREAD

**Discussion by BingulanMenace** [ScratchAddons/ScratchAddons#7408] [Ideas]
I think it would be nice if we could automatically disable all of the addons in the settings menu instead of having to manually disable each one or uninstall and reinstall the addon.

I think the button to clear all addons should be next to the "View settings file" button.

**Comment by CST1229** [comment]
Just disable the extension.

**Reply by WorldLanguages** [reply]
#6570
